src/netbase.cpp

```diff
@@ -137,6 +166,26 @@ bool LookupHost(const char *pszName, CNetAddr& addr, bool fAllowLookup)
     return true;
 }
 
+/**
+ * Resolve a service string to its corresponding service.
+ *
+ * @param pszName The string representing a service. Could be a name or a
```

Comment:
nit: maybe align name params and description comments to ease readability, at least some functions in others parts of code are doing it

Reply:
Not sure what exact alignment you mean, but my `netaddress` comments have been aligned like this as well. Lmk!

Reply:
Was thinking something likely to `BroadcastTransaction` in `transaction.h`, but that's a nit, doesn't matter!